.github/actions/e2e/install-prometheus/values.yaml

```diff
@@ -47,10 +47,10 @@ prometheus:
     resources:
       requests:
         cpu: 1
-        memory: 5Gi
```

Comment:
Just checking, but this is for one of the prometheus stateful sets, not the daemonset right?

Reply:
Yes, it's for stateful set